homeassistant/components/netatmo/const.py

```diff
@@ -27,17 +30,21 @@
 EVENT_HUMAN = "human"
 EVENT_ANIMAL = "animal"
 EVENT_VEHICLE = "vehicle"
-
-EVENT_BUS_PERSON = "netatmo_person"
-EVENT_BUS_MOVEMENT = "netatmo_movement"
-EVENT_BUS_HUMAN = "netatmo_human"
-EVENT_BUS_ANIMAL = "netatmo_animal"
-EVENT_BUS_VEHICLE = "netatmo_vehicle"
-EVENT_BUS_OTHER = "netatmo_other"
+EVENT_OUTDOOR = "Outdoor"
```

Comment:
Is this still needed to be added ?

Reply:
As a constant? No, it's probably not going to be reused elsewhere.